keel-api/src/main/resources/swagger-api.yml

```diff
@@ -0,0 +1,33 @@
+swagger.api:
```

Comment:
So we couldn't use Swagger code annotations to specify the info that's in this file?

Reply:
I'd rather have it in this file.